### ISSUE TITLE
[Paper Note] OpenThoughts-Agent: Data Recipes for Agentic Models, Negin Raoof+, arXiv'26, 2026.06

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#5866]
# URL
- https://arxiv.org/abs/2606.24855
# Authors
  - Negin Raoof
  - Richard Zhuang
  - Marianna Nezhurina
  - Etash Guha
  - Atula Tejaswi
  - Ryan Marten
  - Charlie F. Ruan
  - Tyler Griggs
  - Alexander Glenn Shaw
  - Hritik Bansal
  - E. Kelly Buchanan
  - Artem Gazizov
  - Reinhard Heckel
  - Chinmay Hegde
  - Sankalp Jajee
  - Daanish Khazi
  - Emmanouil Koukoumidis
  - Xiangyi Li
  - Hange Liu
  - Shlok Natarajan
  - Harsh Raj
  - Nicholas Roberts
  - Ethan Shen
  - Nishad Singhi
  - Michael Siu
  - Ashima Suvarna
  - Hanwen Xing
  - Patrick Yubeaton
  - Robert Zhang
  - Leon Liangyu Chen
  - Xiaokun Chen
  - Steven Dillmann
  - Saadia Gabriel
  - Xunyi Jiang
  - Anurag Kashyap
  - Boxuan Li
  - Yein Park
  - Minh Pham
  - Sujay Sanghavi
  - Lin Shi
  - Ke Sun
  - Yixin Wang
  - Zhiwei Xu
  - Erica Zhang
  - Siyan Zhao
  - Wanjia Zhao
  - Jenia Jitsev
  - Alex Dimakis
  - Benjamin Feuer
  - Ludwig Schmidt
# Abstract
  - Agentic language models dramatically expand the applications of AI yet little is publicly known about how to curate training data for broadly capable agents. Existing open efforts such as SWE-Smith, SERA, and Nemotron-Terminal typically target a single benchmark, leaving open the question of how to train models that generalize across diverse agentic tasks. The OpenThoughts-Agent (OT-Agent) project addresses this gap with a fully open data curation pipeline for training agentic models. We conduct more than 100 controlled ablation experiments to systematically investigate each stage of the pipeline, yielding insights on the importance of task sources and diversity. We then assemble a training set of 100K examples from our pipeline and fine-tune Qwen3-32B on this dataset, which yields an average accuracy of 44.8% across seven agentic benchmarks and a 3.9 percentage point improvement over the strongest existing open data agentic model (Nemotron-Terminal-32B, 40.9%). Moreover, our training data exhibits strong scaling properties, outperforming alternative open datasets at every training set size in compute-controlled comparisons. We publicly release our training sets, data pipeline, experimental data, and models at openthoughts.ai to support future open research on agentic model training.
# Translation (by gpt-5-nano)
- エージェント性を持つ言語モデルはAIの応用範囲を大幅に拡張する一方で、広く能力を持つエージェントの訓練データをどうキュレーションするかについて公に知られていることはほとんどありません。SWE-Smith、SERA、Nemotron-Terminalといった既存の公開プロジェクトは通常、単一のベンチマークを対象としており、多様なエージェント的タスクにまたがって一般化できるモデルを訓練するにはどうすべきかという問題は残っています。OpenThoughts-Agent（OT-Agent）プロジェクトは、エージェント性を持つモデルの訓練のための完全に公開されたデータキュレーションパイプラインでこのギャップに対処します。私たちはパイプラインの各段階を体系的に調査するために100件を超える統制済みアブレーション実験を実施し、タスクソースと多様性の重要性について洞察を得ました。続いて、パイプラインから10万件の例を訓練データとして取りまとめ、このデータセットでQwen3-32Bをファインチューニングしたところ、七つのエージェント的ベンチマークで平均正解率は44.8％となり、最も強力な既存のオープンデータエージェントモデル（Nemotron-Terminal-32B、40.9％）より3.9ポイント改善されました。さらに、我々の訓練データは強いスケーリング特性を示し、計算リソースを制御した比較で、訓練セットサイズのいずれの段階においても他のオープンデータセットを上回っています。我々は訓練データセット、データパイプライン、実験データ、そしてモデルをopenthoughts.aiで公開し、エージェント的モデル訓練に関する今後のオープン研究を支援します。
# Summary (by gpt-4o-mini)
- エージェント性を持つ言語モデルの訓練データを公開するOT-Agentプロジェクトが、データキュレーションの重要性に対応。100件以上のアブレーション実験を通じ、タスクソースと多様性の影響を分析。10万件の例を使用してQwen3-32Bをファインチューニングし、七つのベンチマークで平均正解率を44.8％に向上。訓練データは強いスケーリング特性を示し、他のオープンデータセットを超える結果を記録。データセットやパイプラインはopenthoughts.aiで公開し、今後の研究を支援。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/iscienceluvr/status/2069643721155793114?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
著者ポスト:https://x.com/richardz412/status/2069827815403557287?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q
